cmd/kpod/common.go

```diff
@@ -133,3 +136,19 @@ func validateFlags(c *cli.Context, flags []cli.Flag) error {
 	}
 	return nil
 }
+
+// getCachedCreds returns the DockerAuthConfig of a registry if stored in the authfile
+func getCachedCreds(registry string, sc *types.SystemContext) (*types.DockerAuthConfig, error) {
+	if registry == "" {
```

Comment:
Can this ever happen? I don’t know the code but I couldn't see how from a quick look.